sig-contributor-experience/README.md

```diff
@@ -138,4 +138,28 @@ Creates and maintains tools and automation for Kubernetes Slack.
 [subproject-definition]: https://github.com/kubernetes/community/blob/master/governance.md#subprojects
 <!-- BEGIN CUSTOM CONTENT -->
 
+## Relevant Presentations
+
+This is a collection of presentations from our SIG members around contributing to Kubernetes, feel free to PR in content here: 
+
+## Community Updates
+
+
+- July 2020 Community Update - [Recording] (https://youtu.be/J3O8fXTm3HE?t=148) | [Slides](https://docs.google.com/presentation/d/1BS7liUs1WDSPnpNXIHRtRSKhGzK9usS5qzpry_9YU-I/edit?usp=sharing)
```

Comment:
```suggestion
- July 2020 Community Update - [Recording](https://youtu.be/J3O8fXTm3HE?t=148) | [Slides](https://docs.google.com/presentation/d/1BS7liUs1WDSPnpNXIHRtRSKhGzK9usS5qzpry_9YU-I/edit?usp=sharing)
```
caught a space in there XD

Reply:
My change supercedes this and has the date.